src/app/about/about.component.spec.ts

```diff
@@ -46,6 +46,12 @@ describe('Component: About', () => {
     expect(footerNavbar).toBeTruthy();
   });
 
+  it('should have a footer element', () => {
+    let compiled = fixture.debugElement.nativeElement;
```

Comment:
Use ```const```

Reply:
@simsausaurabh should I change all `let` instances to `const` instance or just the instances I added?

Reply:
Please change all instances in this file and check if the tests pass correctly.

Reply:
@simsausaurabh I changed all the `let` instances to `const` instance except on 8th and 9th line. And it is passing `ng test` successfully.

Reply:
@simsausaurabh Please update me. How should I proceed?

Reply:
@simsausaurabh @praveenojha33  Please update. I need to proceed in this.